mlir/lib/Dialect/Rock/Transforms/GridwiseGemmToBlockwise.cpp

```diff
@@ -930,24 +930,36 @@ struct GridwiseAttentionAccelRewritePattern
 
   // This fuction creates interrim register buffers to store data in once
   // loaded from the LDS before accelerator intrinsics are called
-  std::tuple<Value, Value>
+  std::tuple<SmallVector<Value>, SmallVector<Value>>
```

Comment:
High-level question: how'll this integrate with pipelining/multibuffering?

Reply:
Moved to be repeat x ... memref instead now. so its more "compatible" with multibuffering